martus-client/source/org/martus/client/swingui/jfx/landing/general/ManageServerSyncRecordsController.java

```diff
@@ -216,7 +214,14 @@ protected void failed()
 		@Override
 		protected Vector call() throws Exception
 		{
-			serverMyDrafts = getServerMyDrafts();
+			try
```

Comment:
LoadServerData should have an abstract method loadData() throws Exception,  and LoadServerData would override the call method with the try catch.  This will eliminate the 4 try catch calls in the node classes.